packages/material-ui/src/Typography/Typography.js

```diff
@@ -97,14 +75,27 @@ const useUtilityClasses = (styleProps) => {
   return composeClasses(slots, getTypographyUtilityClass, classes);
 };
 
+const colorTransformations = {
```

Comment:
Should we still support these values in v5 and just throw a deprecation warning if they are used?

Reply:
No specific preference for doing the deprecation in v5 or v4.